ecosystem/ffmpeg_plugin/mtl_st20p_rx.c

```diff
@@ -178,7 +185,17 @@ static int mtl_st20p_read_packet(AVFormatContext* ctx, AVPacket* pkt) {
   struct st_frame* frame;
 
   dbg("%s(%d), start\n", __func__, s->idx);
-  frame = st20p_rx_get_frame(s->rx_handle);
+
+  if (0 == s->frame_counter) {
+    for (int i = 1; i <= 10; i++) {
```

Comment:
If you insist on to use the WA, please add some comment explain why it need a WA here, and adding a config option instead of the hardcode `10`.

Reply:
That is a good suggestion, implemented it with this change